cumulus/polkadot-omni-node/lib/Cargo.toml

```diff
@@ -28,10 +28,14 @@ docify = { workspace = true }
 # Local
 jsonrpsee = { features = ["server"], workspace = true }
 parachains-common = { workspace = true, default-features = true }
+scale-info = { workspace = true }
+subxt = { workspace = true, features = ["native"] }
```

Comment:
At least this dependency is unnecessary, you should use `subxt_metadata::Metadata`.

Reply:
Testing this now, will push a change after it is done.

Reply:
Yep, works great, thanks! Done here: [c6e9f8c](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/c6e9f8c51b17082a9ed627531e822784eae825a1)